elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
```

Comment:
you're right! ill do that.  
head.html is still in template form - it's complete, so i probably shouldn't have committed that yet.
title.html is the one i thought was ready. but yeah ill get rid of those comments.